### PR TITLE
configure a default value as function param for getCommandOrCtrlKey

### PR DESCRIPTION
**Pull request by abhijithhegde**
#1 function will return default values or custom values on call

### CODE REVIEW

**Comment by abhijithhegde** [comment]
@devcer could you please merge this?

File: command-or-ctrl-key/src/index.ts [hunk 1]
```diff
@@ -2,12 +2,22 @@
  * Get the appropriate keyboard shortcut key based on the user's platform.
  * Returns '⌘' for macOS and 'Ctrl' for other operating systems.
  */
-export const getCommandOrCtrlKey = () => {
-  const userAgent = navigator.userAgent.toLowerCase();
+export const getCommandOrCtrlKey = ({
```
Comment (devcer) [inline command-or-ctrl-key/src/index.ts]: * create a `src/constants.ts` file
* Define a `DEFAULT_CONFIG = {.. your config}` 
* Import and pass it as a config `export const getCommandOrCtrlKey = (config = DEFAULT_CONFIG) =>{`
* update related tests as well

File: command-or-ctrl-key/src/index.ts [hunk 2]
```diff
@@ -2,12 +2,22 @@
...
-    return "Ctrl"; // Other operating systems
-  }
+	if (userAgent.includes("mac")) {
+		return mac; // macOS
+	} else {
+		return others; // Other operating systems
+	}
 };
+//call the function with default values
+const defaultKey = getCommandOrCtrlKey();
```
Comment (devcer) [inline command-or-ctrl-key/src/index.ts]: remove from here and test these using unit tests.

**Review by devcer — CHANGES_REQUESTED** [review]
Need some changes

**Comment by abhijithhegde** [comment]
Hey, I have updated the code! please check once

**Review by devcer — APPROVED** [review]

**Pull request merged**